sdk/go/pulumi/config/get.go

```diff
@@ -90,6 +92,17 @@ func GetInt64(ctx *pulumi.Context, key string) int64 {
 	return 0
 }
 
+// GetObject attempts to load an optional configuration value by its key into the specified output variable.
+func GetObject(ctx *pulumi.Context, key string, output interface{}) error {
```

Comment:
We might want to have this return `(bool, error)` and have the first return value indicate whether or not the value was found. We probably want the same pattern for the other APIs, so we don't have to do this now.

Reply:
I see. As it is for primitive types, the user has to check the output value, but this isn't a reliable signal especially if you could be expecting the zero value.